vignettes/example.Rmd

```diff
@@ -0,0 +1,2197 @@
+trials, are shown. Note that you can select the plots by changing the
+`type` argument of `plot`, which by default is `type =
+  c("trajectory", "dosesTried")`.
+
+Second, we can summarize the simulation results. Here again we have to supply a
+true dose-toxicity function. We take the same (`myTruth`) as above:
+
+```{r sim-summary}
+summary(mySims,
+  truth = myTruth
+)
+```
+
+Note that sometimes the observed toxicity rate at the dose most often selected
+(here 20 mg) is not available, because it can happen that no patients were
+actually treated that dose during the simulations. (Here it is available.) This
+illustrates that the MTD can be selected based on the evidence from the data at
+other dose levels -- which is an advantage of model-based dose-escalation
+designs.
+
+Now we can also produce a plot of the summary results, which gives a bit more
+detail than the textual summary we have just seen:
+
+```{r sim-sum-plot, fig=TRUE, fig.width=6}
+simSum <- summary(mySims,
+  truth = myTruth
+)
+print(plot(simSum))
+```
+
+The top left panel shows the distribution of the sample size across the
+simulated trials. In this case the trials had between 15 and 21 patients. The
+top right panel shows the distribution of the final MTD estimate / recommended
+dose across the simulated trials. The middle left panel shows the distribution
+across the simulations of the DLT proportions observed in the patients dosed.
+Here in most trials between 20 and 30% of the patients had DLTs.
+The middle right panel shows the distribution across simulations of the number
+of patients treated above the target toxicity window (here we used the default
+from 20% to 35%). Finally, in the bottom panel we see a comparison
+of the true dose-toxicity curve (black) with the estimated dose-toxicity curves,
+averaged (continuous red line) across the trials and with 95% credible interval
+across the trials. Here we see that the steep true dose-toxicity curve is not
+recovered by the model fit.
+
+If we find that e.g. the top right plot with the distribution of the final
+selected doses is too small and shows not the right x-axis window, we can only
+plot this one and add x-axis customization on top: (see the `ggplot2`
+documentation for more information on customizing the plots)
+
+```{r sim-sum-plot2, fig=TRUE}
+dosePlot <- plot(simSum, type = "doseSelected") +
+  scale_x_continuous(breaks = 10:30, limits = c(10, 30))
+print(dosePlot)
+```
+
+Some further examples will be given for simulations using the `TDDesign` and the `TDsamplesDesign` classes. For illustration purpose, we will generate only 10 trial outcomes.
+
+```{r DLTSim-run}
+DLTSim <- simulate(DLTdesign,
+  args = NULL,
+  truth = TrueDLT,
+  nsim = 10,
+  seed = 819,
+  parallel = FALSE
+)
+```
+
+The above is an example when no MCMC sampling is involved and we have another example below for simulation when MCMC sampling is involved:
+
+```{r DLTsampSim-run,results=FALSE}
```

Comment:
can we suppress the (too many) plots here?

Reply:
Maybe it is easy to suppress then that would be preferred

Reply:
The `%%\Vignette...` stuff comes from the standard vignette template created by `devtools::use_vignette()`, so I guess it's needed.  The setting of some `options` was in the original Rnw file.  I don't think it's needed in the Rmd file.  I didn't experiment to check, but I can if you wish.

Reply:
yeah it would be good to clean up thanks